Extensions/Video/videoruntimeobject.js

```diff
@@ -0,0 +1,241 @@
+/**
+ * A video object doing showing a text on screen
+ *
+ * @memberof gdjs
+ * @class VideoRuntimeObject
+ * @extends RuntimeObject
+ */
+gdjs.VideoRuntimeObject = function(runtimeScene, objectData) {
+  // Always call the base gdjs.RuntimeObject constructor.
+  gdjs.RuntimeObject.call(this, runtimeScene, objectData);
+
+  // Load any required data from the object properties.
+  this._property1 = objectData.content.property1;
+
```

Comment:
Remember to initialize opacity to its default value (see my other comments, opacity should live here, in the object): `this._opacity = 255;`